pypeit/scripts/show_2dspec.py

```diff
@@ -133,7 +133,15 @@ def main(args):
     slit_ids = [edgetrace.get_slitid(shape, tslits_dict['slit_left'], tslits_dict['slit_righ'],
                                      ii)[0] for ii in range(tslits_dict['slit_left'].shape[1])]
     # Show the bitmask?
-    mask_in = mask if args.showmask else None
+    if args.showmask:
```

Comment:
Why are you importing the ImageBitMask here and then passing it in. You could just do this in show_image, and apply the bitmask if the mask_in is passed. Not sure why you need to pass this instance here rather than just creating it in show_image. Is the issue that you just want to create it once?

Reply:
I had tried to use this functionality.  
Then failed. Am pretty sure this 
preserves the original intent.